posthog/api/test/test_organization_feature_flag.py

```diff
@@ -256,12 +266,17 @@ def test_copy_feature_flag_update_override_deleted(self):
         existing_deleted_flag.usage_dashboard = usage_dashboard
         existing_deleted_flag.save()
 
+        # Experiments restrict deleting soft-deleted flags
```

Comment:
cc: @jurajmajerik we shouldn't really get to this stage because the frontend prevents soft-deleting experiment related flags now, but just fyi